pages/design/rule-design.md

```diff
@@ -39,34 +73,64 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input)
+
 ### Authors
 
 Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`.
```

Comment:
worth adding a note, that if the contributor does not exist in the package.json, the PR should also amend the contributors list if a new author is intended to be added.